CHANGES.rst

```diff
@@ -1,17 +1,22 @@
 Changelog
 =========
 
-Version 0.3.0 [unreleased]
+Version 0.3.0 [2020-07-03]
 --------------------------
 
-WIP
-
-- `organization owners <https://github.com/openwisp/openwisp-users#organization-owners>`_:
-  [admin] Added support for organization owners
-- [admin] Made organization owners read-only to non-superusers
-- `organization permissions <https://github.com/openwisp/openwisp-users#organization-permissions>`_:
-  [admin] Allowed administrator role to access organization admin
-- [model] Added `user permission helpers <https://github.com/openwisp/openwisp-users#permissions-helpers>`_
+- [admin] Added default owner to each organization
```

Comment:
It should be relevant to mention these features:

- [feature] Added DRF org membership permission classes
- Replaced organizations_pk with organizations_dict
- Organization membership related methods